rpc/client/evidence_test.go

```diff
@@ -171,3 +168,20 @@ func TestBroadcastEmptyEvidence(t *testing.T) {
 		assert.Error(t, err)
 	}
 }
+
+func waitForBlock(ctx context.Context, t *testing.T, c client.Client, height int64) {
```

Comment:
I left this in a different thread that might have gotten hidden, but this function will, I think, always run for > 200ms, because we wait *first* and then check. Previous versions checked the height (possibly returned) and then waited 200ms. 

If this is a timer initialized to 0, and we reset it each time to 200ms we'll get the previous behavior, which might be more efficient in some cases.